src/storage/tracked.rs

```diff
@@ -118,6 +118,7 @@ where
     /// impl Component for Comp {
     ///     // Will use `DenseVecStorage` by default.
     ///     type Storage = TrackedStorage<Self>;
+    ///     type Metadata = ();
```

Comment:
So `TrackedStorage` doesn't work with `Metadata`?

Reply:
It could work, I just haven't ported it over like the FlaggedStorage yet.

Reply:
I don't think we should proceed until `TrackedStorage` is ported